packages/frontend/src/components/navigation/ConnectLedger.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import CheckCircleIcon from '../svg/CheckCircleIcon';
+import LedgerSmall from '../svg/LedgerSmall';
+
+const ConnectLedgerButton = styled.div`
+    background: #F0F0F1;
+    border-radius: 4px;
+    color: #72727A;
+    font-weight: 600;
+    padding: 14px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    cursor: pointer;
+
+    svg {
+        width: 16px;
+        height: 16px;
+        margin-right: 8px;
+    }
+`;
+
+export default ({
+    connectLedger,
+    ledgerConnectionAvailable,
+    showConnectLedgerButton
+}) => {
+    return (
+        <>
+            {showConnectLedgerButton &&
```

Comment:
Seems odd to render an empty component based on a prop. What if we removed this prop and had the parent only render this when `showConnectLedgerButton` is `true`?

Reply:
Right, refactored it to return `null` - this is the pattern that we use in a few places - the idea is that the parent component doesn't have to care about showing/not showing the component - everything related to this component is inside it. Are you ok with this solution?

Reply:
But the parent _does_ care because it knows the value of `showConnectLedgerButton`. To me it's more clear in the parent to know whether or not a child component is rendered. Child components shouldn't take a flag that prevents the component from rendering, it should just be a dumb component that always renders and the parent decides whether or not to render it.

Reply:
I see your point. Actually, we don't need to pass these props, this component can handle all necessary data - it would be much more effective:
https://github.com/near/near-wallet/pull/2595/commits/8e3d121d6c5d1a038aa8445fc5689eb6a8fa2a85